src/styles/ingredients.scss

```diff
@@ -0,0 +1,182 @@
+
+  @include tablet {
+    position: relative;
+    margin-bottom: 130px;
+  }
+
+  @include desktop {
+    max-width: 1280px;
+    margin-bottom: 201px;
+  }
+
+  @include big-desktop {
+    max-width: 1440px;
+  }
+
+  &__title {
+    position: relative;
+    max-width: 280px;
+    margin-bottom: 20px;
+
+    @include tablet {
+      margin-bottom: 310px;
+    }
+
+    @include desktop {
+      max-width: 380px;
+      margin-bottom: 287px;
+    }
+  }
+
+  &__blok {
+    display: block;
+    background: url(../images/svg/bank-mobile.svg) no-repeat right/contain;
+    width: 100%;
+    height: 397px;
+    position: relative;
+    margin-bottom: 30px;
+    right: -20px;
+
+    @include tablet {
+      position: absolute;
+      background: url(../images/svg/bank-tablet.svg) no-repeat right/contain;
+      bottom: 35%;
+      right: 0;
+    }
+
+    @include desktop {
+      position: absolute;
+      width: 600px;
+      background: url(../images/svg/bank.svg) no-repeat right/contain;
+      bottom: 35%;
+      right: 50px;
+    }
+
+    @include big-desktop {
+      right: 120px;
+    }
+  }
+
+  &__list {
+    display: flex;
+    flex-direction: column;
+    gap: 52px;
+    text-align: right;
+    white-space: nowrap;
+    position: absolute;
+    padding: 33px 190px 0 0;
+    right: 10px;
+
+    @include tablet {
+      padding: 33px 418px 0 0;
+    }
+
+    @include desktop {
+      right: 80px;
+    }
+  }
+
+  &__description {
+    margin-bottom: 110px;
+
+    @include tablet {
+      margin-bottom: 130px;
+    }
+
+    @include desktop {
+      padding-left: 100px;
+      margin-bottom: 218px;
+    }
+  }
+
+  &__advantages {
+    text-align: center;
+    display: flex;
+    flex-direction: column;
+    gap: 50px;
+    max-width: 280px;
+    margin: auto;
+
+    @include tablet {
+      margin-bottom: 130px;
+      max-width: 100%;
+      flex-direction: row;
+      align-items: baseline;
+      justify-content: space-between;
+    }
+
+    @include desktop {
+      padding-left: 100px;
+      gap: 45px;
+    }
+  }
+
+  &__advantages-item {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    gap: 20px;
+    line-height: 130%;
+    letter-spacing: 0.7px;
+    font-size: 14px;
+    text-transform: uppercase;
+
+    @include tablet {
+      max-width: 218px;
+    }
+
+    @include desktop {
+      max-width: 380px;
+    }
+  }
+
+  &__advantages-snail {
+    width: 62px;
+    height: 60px;
+
+    @include tablet {
+      width: 80px;
+      height: 60px;
+    }
+  }
+
+  &__advantages-recast {
+    width: 100px;
+    height: 84px;
+
+    @include tablet {
+      width: 80px;
+      height: 67.87px;
+    }
+
+    @include desktop {
+      width: 99px;
+      height: 84px;
+    }
+  }
+
+  &__advantages-leaf {
+    width: 79px;
+    height: 74px;
+
+    @include tablet {
+      width: 80px;
+      height: 74.93px;
+    }
+
+    @include desktop {
+      width: 79px;
+      height: 74px;
+    }
+  }
+
+  &__left {
```

Comment:
The 'left' property is used but the 'position' property is not set. Make sure to set the 'position' property to something other than 'static' for the 'left' property to have an effect.